docs/license.rst

```diff
@@ -13,20 +13,38 @@
 Using and Citing SORA
 =====================
 
-Terms of Use
-------------
+Citing
+------
 
 
 SORA is free software and we kindly ask users that make use of it (or part of it), especially in 
-projects, if that results in scientific publications, to include the following acknowledgment 
-statement:
+projects, if that results in scientific publications, to include the following citation:
 
 ::
 
-    “This research made use of SORA, a python package for stellar occultations reduction and 
-    analysis, developed with the support of ERC Lucky Star and LIneA/Brazil.”
+    Gomes-Júnior et al. (2022). SORA: Stellar occultation reduction and analysis.
+    MNRAS, Volume 511, Issue 1, March 2022, Pages 1167–1181
+    DOI: 10.1093/mnras/stac032
 
 
+The scientific documentation of SORA is described in the paper available on Monthly Notices
+of the Royal Astronomical Society, which can be accessed on the following links:
+   * |link_mnras|;
```

Comment:
Spaces (L32-L34) before '*' should be removed (generates an error on compiling the documentation)